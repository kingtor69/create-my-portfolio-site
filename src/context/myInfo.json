{
    "navigation": {
        "name": "KingTorCodes",
        "menu": [
            {
                "text": "Home",
                "url": "/",
                "target":""
            },
            {
                "text": "About",
                "url": "/about",
                "target":""
            },
            {
                "text": "Github",
                "url": "https://github.com/kingtor69",
                "target":"_blank"
            },
            {
                "text": "LinkedIn",
                "url": "https://www.linkedin.com/in/kingtor",
                "target":"_blank"
            }
        ]
    },
    "main": {
        "greeting": "My name is Tor Kingdon and I tell stories with code.",
        "blurb":"I've been working in the entertainment business since I was a kid (specifically sound for TV & film) and have decided to use my computer skills to write code instead of making funny noises.",
        "profileImage": { 
            "url":"https://raw.githubusercontent.com/kingtor69/create-my-portfolio-site/main/src/assets/TorHeadshotWithCello400x400.jpeg",
            "altText":"Tor in front of Carnegie Hall"
        },
        "image": {
            "url": "",
            "altText": "temp"
        }
    },
    "work":{
        "categoryName":"Selected Work",
        "projects":[
            {
                "projectName": "Saddle-Time",
                "description":"Bicycle-friendly bicycle directions. It was programmed with Python, HTML, CSS, and JavaScript. I also used the frameworks/libraries Flask, WTForms, Jinja2, Bootstrap, jQuery and plugins from Mapbox & Select2. It uses APIs from Mapbox, Mapquest, and Open Weather Service.",
                "githubUrl":"https://github.com/kingtor69/saddle-time",
                "projectUrl":"https://saddle-time.herokuapp.com/",
                "image": {
                    "url": "https://raw.githubusercontent.com/kingtor69/create-my-portfolio-site/main/src/assets/SaddleTimeScreenshot.png",
                    "altText": "Saddle-Time screenshot"
                }
            },
            {
                "projectName": "Tor Boggle",
                "description":"My down-n-dirty take on the classic dice/word game using Python/Flask.",
                "githubUrl":"https://github.com/kingtor69/flask-boggle",
                "projectUrl":"http://tor-boggle.herokuapp.com/",
                "image": {
                    "url": "https://raw.githubusercontent.com/kingtor69/create-my-portfolio-site/main/src/assets/TorBoggle.png",
                    "altText": "Tor Boggle screenshot"
                }
            },
            {
                "projectName": "The Milonga Project",
                "description":"I produced an “illustrated audio play” with my partner and a wrtier/director we’ve worked with before, and I made our website. The backend is still in development using Node JS and the Strapi library.",
                "githubUrl":"https://github.com/kingtor69/themilongaproject.com",
                "projectUrl":"https://themilongaproject.com/",
                "image": {
                    "url": "https://raw.githubusercontent.com/kingtor69/create-my-portfolio-site/main/src/assets/themilongaprojectScreenShot.png",
                    "altText": "The Milonga Project screenshot"
                }
            }
        ],
        "buttonNames":{
            "primary":"Github",
            "secondary":"Site"
        }
    },
    "about": {
        "categoryName":"About",
        "statement": "I've been using computers since the Commodore PET, and programming since PASCAL was pretty hot stuff. I learned PASCAL first on my family's Apple ][+ with the RAM expansion to 64k! I graduated with a concentration in Computer Science from Kalamazoo College where I learned a variety of database skills along with the basic structure applicable to all programming languages. Learning how to 'think like a computer' has served me very well in everything I've done, from temping in offices to mixing feature films.",
        "skills":{
            "categoryName":"Skills",
            "items":[
                {
                    "type":"Languages",
                    "items":["JavaScript", "HTML", "CSS","Python", "SQL"]
                },
                {
                    "type":"Libraries and Frameworks",
                    "items":["React", "Express", "Flask", "Bootstrap", "WTForms", "Jinja2", "jQuery"]
                },
                {
                    "type":"Fun",
                    "items":["hockey center", "black diamond skier", "cyclist (roadie)", "pretty good bridge partner"]
                }
            ]
        },
        "workExperience": {
            "categoryName":"Work Experience",
            "work": [
                {
                    "company":"Springboard",
                    "position":"Software Engineering Student/Fellow",
                    "date":"October 2020 - January 2022",
                    "description":"I have rediscovered a love for computers that has been with me since my first experience at a computer drop-in. I'm working primarily in web app development (full stack) but am also looking to merge my audio post-production experience with coding."
                },
                {
                    "company":"Hear Kitty Studios",
                    "position":"Co-owner / Chief Sound Guy",
                    "date":"October 2002 - present",
                    "description":"Kathleen Edwards, M.P.S.E. and I founded Hear Kitty Studios to combine our talents and offer anyone interested in producing television, radio or podcast content the highest quality sound possible. We are passionate about post-production sound, crafting custom soundscapes to make every project we work on sound its best."
                }
            ]
        },
        "education":{
            "categoryName":"Education",
            "items": [
                {
                    "name":"Springboard",
                    "curriculum":"Full Stack Programming",
                    "date":"October 2020 - January 2022",
                    "description":"The dream job that I'm working towards would merge my sound experience with my love for code."
                },
                {
                    "name":"Kalamazoo College",
                    "curriculum":"Bachelor of Arts",
                    "date":"September 1987 - June 1991",
                    "description":"Graduated with Honors in Theatre Arts & Communication with a concentration with Computer Science"
                }
            ]
        },
        "certifications":{
            "categoryName":"Certifications",
            "items":[]
        }
    },
    "other":{
        "images":[
            {"image":{
                "url":"https://raw.githubusercontent.com/kingtor69/create-my-portfolio-site/main/src/assets/soundAndCodeDesk.jpg",
                "altText":"sound guy who codes",
                "description":"My desk with Pro Tools, VS Code, Terminal window, web browser, and sound design tools."
                }
            }
        ]
    }
}