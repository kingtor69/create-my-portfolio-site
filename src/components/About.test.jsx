import React from 'react';
import { render } from '@testing-library/react';
import About from './About';
import myInfo from '~/Dropbox/kingtorcodesData/context/myInfo.json';
import MyInfoContext from '~/Dropbox/kingtorcodesData/context/MyInfoContext';

// smoke test
it('renders without crashing', () => {
	render(
		<MyInfoContext.Provider value={myInfo}>
			<About />
		</MyInfoContext.Provider>
	);
});

// snapshot test
it('matches snapshot', () => {
	const { asFragment } = render(
		<MyInfoContext.Provider value={myInfo}>
			<About />
		</MyInfoContext.Provider>
	);
	expect(asFragment()).toMatchSnapshot();
});
