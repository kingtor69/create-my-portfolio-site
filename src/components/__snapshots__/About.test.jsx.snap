// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root makeStyles-container-8 MuiContainer-maxWidthMd"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation2 MuiPaper-rounded"
    >
      <div
        class="makeStyles-category-9"
      >
        <h4
          class="MuiTypography-root MuiTypography-h4"
        >
          About
        </h4>
        <div
          class="makeStyles-categoryBody-10"
        >
          <h6
            class="MuiTypography-root makeStyles-aboutStatement-14 MuiTypography-subtitle1"
          >
            I've been using computers since the Commodore PET, and programming since PASCAL was pretty hot stuff. I learned PASCAL first on my family's Apple ][+ with the RAM expansion to 64k! I graduated with a concentration in Computer Science from Kalamazoo College where I learned a variety of database skills along with the basic structure applicable to all programming languages. Learning how to 'think like a computer' has served me very well in everything I've done, from temping in offices to mixing feature films.
          </h6>
        </div>
        <div
          class="MuiPaper-root MuiCard-root makeStyles-aboutImage-11 MuiPaper-elevation1 MuiPaper-rounded"
        >
          <div
            class="MuiCardMedia-root makeStyles-media-12"
            style="background-image: url(https://raw.githubusercontent.com/kingtor69/create-my-portfolio-site/main/src/assets/soundAndCodeDesk.jpg);"
            title="sound guy who codes"
          />
        </div>
        <span
          class="MuiTypography-root makeStyles-mediaDescription-13 MuiTypography-caption MuiTypography-colorTextSecondary MuiTypography-alignCenter MuiTypography-displayBlock"
        >
          My desk with Pro Tools, VS Code, Terminal window, web browser, and sound design tools.
        </span>
      </div>
      <div
        class="MuiContainer-root MuiContainer-maxWidthSm"
      >
        <hr
          class="MuiDivider-root"
        />
      </div>
      <div
        class="makeStyles-category-9"
      >
        <h4
          class="MuiTypography-root MuiTypography-h4"
        >
          Skills
        </h4>
        <div
          class="makeStyles-categoryBody-10"
        >
          <div
            class="0"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom"
            >
              Languages
            </h6>
            <p
              class="MuiTypography-root MuiTypography-body1"
            >
              JavaScript, HTML, CSS, Python, SQL
            </p>
            <br />
          </div>
          <div
            class="1"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom"
            >
              Libraries and Frameworks
            </h6>
            <p
              class="MuiTypography-root MuiTypography-body1"
            >
              React, Express, Flask, Bootstrap, WTForms, Jinja2, jQuery
            </p>
            <br />
          </div>
          <div
            class="2"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom"
            >
              Fun
            </h6>
            <p
              class="MuiTypography-root MuiTypography-body1"
            >
              hockey center and ref, black diamond skier, cyclist (roadie), passable bridge partner
            </p>
            <br />
          </div>
        </div>
      </div>
      <div
        class="MuiContainer-root MuiContainer-maxWidthSm"
      >
        <hr
          class="MuiDivider-root"
        />
      </div>
      <div
        class="makeStyles-category-9"
      >
        <h4
          class="MuiTypography-root MuiTypography-h4"
        >
          Work Experience
        </h4>
        <div
          class="makeStyles-categoryBody-10"
        >
          <div>
            <h6
              class="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom"
            >
              Springboard
            </h6>
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
            >
              Software Engineering Student/Fellow
            </p>
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
            >
              October 2020 - January 2022
            </p>
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
            >
              I have rediscovered a love for computers that has been with me since my first experience at a computer drop-in. I'm working primarily in web app development (full stack) but am also looking to merge my audio post-production experience with coding.
            </p>
          </div>
          <div>
            <h6
              class="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom"
            >
              Hear Kitty Studios
            </h6>
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
            >
              Co-owner / Chief Sound Guy
            </p>
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
            >
              October 2002 - present
            </p>
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
            >
              Kathleen Edwards, M.P.S.E. and I founded Hear Kitty Studios to combine our talents and offer anyone interested in producing television, radio or podcast content the highest quality sound possible. We are passionate about post-production sound, crafting custom soundscapes to make every project we work on sound its best.
            </p>
          </div>
        </div>
      </div>
      <div
        class="MuiContainer-root MuiContainer-maxWidthSm"
      >
        <hr
          class="MuiDivider-root"
        />
      </div>
      <div
        class="makeStyles-category-9"
      >
        <h4
          class="MuiTypography-root MuiTypography-h4"
        >
          Education
        </h4>
        <div
          class="makeStyles-categoryBody-10"
        >
          <div>
            <h6
              class="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom"
            >
              Springboard
            </h6>
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
            >
              Full Stack Programming
            </p>
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
            >
              October 2020 - January 2022
            </p>
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
            >
              The dream job that I'm working towards would merge my sound experience with my love for code.
            </p>
          </div>
          <div>
            <h6
              class="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom"
            >
              Kalamazoo College
            </h6>
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
            >
              Bachelor of Arts
            </p>
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
            >
              September 1987 - June 1991
            </p>
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
            >
              Graduated with Honors in Theatre Arts & Communication with a concentration with Computer Science
            </p>
          </div>
        </div>
      </div>
      <div
        class="MuiContainer-root MuiContainer-maxWidthSm"
      >
        <hr
          class="MuiDivider-root"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
